commit 032a7617e2f99450f6c7d72d3b9c2696df835413
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Mon Dec 3 12:30:24 2018 +0100

    braille: make sure the attribute mask is as long as the text string
    
    otherwise brlapi would get uninitialized bytes and thus output garbage
    at the end of the output.

---
 src/orca/braille.py |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/src/orca/braille.py
+++ b/src/orca/braille.py
@@ -1217,7 +1217,9 @@ def refresh(panToCursor=True,
     if attributeMask:
         submask = attributeMask[startPos:endPos]
     else:
-        submask = '\x00' * (endPos - startPos)
+        submask = ""
+
+    submask += '\x00' * (len(substring) - len(submask))
     if not _brlAPIRunning:
         init(_callback, settings.tty)
     if _brlAPIRunning:
